Cargo.toml

```diff
@@ -8,3 +8,4 @@ clap = { version = "4.0", features = ["derive"] }
 walkdir = "2.3"
 anyhow = "1.0"
 tempfile = "3.12.0"
+colored = "2.0"
```

Comment:
What does this do ?

Reply:
fancy coloring. Don't want everything to be same grey color in the terminal